app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -92,6 +99,14 @@ def extract_pii_from_doc(response)
         )
       end
 
+      # making a new method to avoid breaking the existing flow
+      def save_pii_in_session(pii)
+        flow_session[:pii_from_doc] = pii.merge(
```

Comment:
Looks like this is uncovered, but I also can't find where it is used. I think if we pulled this out we'd fix our coverage woes.